sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -278,18 +291,15 @@ def _from_generated(cls, doc_status):
 
 
 class DocumentTranslationError(object):  # pylint: disable=useless-object-inheritance, R0903
-    """This contains an outer error with error code, message, details, target and an
-    inner error with more descriptive details.
-
-    :ivar code: Enums containing high level error codes. Possible values include:
-     "InvalidRequest", "InvalidArgument", "InternalServerError", "ServiceUnavailable",
-     "ResourceNotFound", "Unauthorized", "RequestRateTooHigh".
-    :vartype code: str
-    :ivar message: Gets high level error message.
-    :vartype message: str
-    :ivar target: Gets the source of the error.
-     For example it would be "documents" or "document id" in case of invalid document.
-    :vartype target: str
+    """This contains the error code, message, and target with descriptive details on why
+    a translation job or particular document failed.
+
+    :ivar str code: The error code. Possible high level values include:
+        "InvalidRequest", "InvalidArgument", "InternalServerError", "ServiceUnavailable",
```

Comment:
not related to this PR, but I noticed the service doesn't have a definition for the `InnerErrorCodes` and because that is what we usually surface to the customer, we can't document what the errors might be.
I was thinking of reaching out to the service team and see if they could add them to swagger. What do you think?

Reply:
I think that's a good idea. I need to check what we did in TA, pretty sure the inner error codes were documented in the swagger.

Reply:
Good point. yeah. TA has them https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L898
I will ping Alex in our Teams' channel